README.md

```diff
@@ -58,18 +68,251 @@ Check out the [configuration part](https://docs.thundra.io/python/configuration-
+
+```sh
+export THUNDRA_AGENT_TRACE_REQUEST_SKIP=True
+```
+
+OR
+
+```python
+import thundra
+
+thundra.configure(
+     options={
+         "config": {
+             "thundra.apikey": <your_thundra_api_key>,
+             "thundra.agent.application.name": <your_application_name>,
+			 "thundra.agent.trace.request.skip": True
+         }
+     }
+)
+```
+
+## Frameworks
+
+The following frameworks are supported by Thundra:
+
+|Framework                               |Supported Version          |Auto-tracing Supported                               |
+|----------------------------------------|---------------------------|-----------------------------------------------------|
+|[AWS Lambda](#aws-lambda)               |All                        |<ul><li>- [x] </li></ul>                             |
+|[Django](#django)                       |`>=1.11`                   |<ul><li>- [x] </li></ul>                             |
+|[Flask](#flask)                         |`>=0.5`                    |<ul><li>- [x] </li></ul>                             |
+|[Fastapi](#fastapi)                     |`>=0.62.0`                 |<ul><li>- [x] </li></ul>                             |
+|[Chalice](#chalice)                     |`>=1.0.0`                  |<ul><li>- [x] </li></ul>                             |
+
+### AWS Lambda
+
+Tracing Lambda functions:
+
+1. Auto trace by using one of the following methods:
+    - [No-Code Change Tracing](#no-code-change-tracing).
+    - [In-Code Configuration Tracing](#in-code-configuration-tracing)
+2. Trace specific endpoints by [Decorator](#decorator)
+
+**NOTES**
+
+- Make sure to choose just one of the methods.
+- For decorator tracing method, just change `@thundra.<framework_wrapper>` by `@thundra.lambda_wrapper`.
+
+### Django
+
+Tracing Django application:
+
+1. Auto trace by using one of the following methods:
+    - [No-Code Change Tracing](#no-code-change-tracing).
+    - [In-Code Configuration Tracing](#in-code-configuration-tracing)
+2. Trace specific endpoints by [Decorator](#decorator)
+
+Code snippet in [In-Code Configuration Tracing](#in-code-configuration-tracing) should be added into `settings.py` file.
+To trace django database processes, following environment variables shall be set to false:
+
+- [No-Code Change Tracing](#no-code-change-tracing):
+    
+    ```sh
+    export THUNDRA_AGENT_TRACE_INTEGRATIONS_DJANGO_ORM_DISABLE=false
+    export THUNDRA_AGENT_TRACE_INTEGRATIONS_RDB_DISABLE=false
+    ```
+
+- [In-Code Configuration Tracing](#in-code-configuration-tracing):
+
+    ```python
+    import thundra
+
+    thundra.configure(
+        options={
+            "config": {
+                "thundra.apikey": <your_thundra_api_key>,
+                "thundra.agent.application.name": <your_application_name>,
+                "thundra.agent.trace.integrations.django.orm.disable": True,
+                "thundra.agent.trace.integrations.rdb.disable": True
```

Comment:
for tracing django database process, this config value should be "False"

Reply:
done.